spec/win32/assets/notes.txt

```diff
@@ -0,0 +1,6 @@
+Steve Balmer pfx file password is 1234
```

Comment:
is this file used anywhere?

Reply:
No, test file created for the specific purpose of testing

Reply:
where in the tests is it used. I must be missing something?